Tests/Runtime/Client/BacktraceAttributeMachineIdTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using Backtrace.Unity.Extensions;
+using Backtrace.Unity.Model;
+using Backtrace.Unity.Tests.Runtime.Client.Mocks;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace Backtrace.Unity.Tests.Runtime.Client
+{
+    class BacktraceAttributeMachineIdTests
+    {
+        [TearDown]
+        public void Cleanup()
+        {
+            PlayerPrefs.DeleteKey(MachineIdStorage.MachineIdentifierKey);
+        }
+
+        [Test]
+        public void TestMachineAttributes_ShouldUseUnityIdentifier_ShouldReturnUnityIdentitfier()
+        {
+            var machineIdStorage = new MachineIdStorageMock();
+
+            var machineId = machineIdStorage.GenerateMachineId();
+
+            Assert.IsFalse(GuidHelper.IsEmptyGuid(machineId));
+        }
+
+        [Test]
+        public void TestMachineAttributes_ShouldUseMac_ShouldReturnNetowrkingIdentifier()
+        {
+            var machineIdStorage = new MachineIdStorageMock(false);
+
+            var machineId = machineIdStorage.GenerateMachineId();
+
+            Assert.IsFalse(GuidHelper.IsEmptyGuid(machineId));
+        }
```

Comment:
Maybe in these two, you should test if the value does not change after clearing PlayerPrefs key?